### PR TITLE
Cleanup formatting

### PR DESCRIPTION
**Pull request by jason-seqera**
This cleans up some ChatGPT-isms.

### CODE REVIEW

File: CONFIGURATION.md
Comment (pditommaso) [inline CONFIGURATION.md:1]: Let's keep `CONFIGURATION.md` lowercase

**Review by pditommaso — COMMENTED** [review]
Apert the file name cases, it looks awesome!

File: CONFIGURATION.md
Comment (jason-seqera) [reply CONFIGURATION.md:1]: @pditommaso, I'm not sure I understand. Is there an example of where something is uppercase and should be lowercase? Thanks!

Comment (pditommaso) [reply CONFIGURATION.md:1]: The file name 😄 

https://github.com/seqeralabs/wave/blob/master/CONFIGURATION.md

**Comment by jason-seqera** [comment]
@pditommaso, this now renames the file to lowercase.

**Review by pditommaso — APPROVED** [review]

**Comment by jason-seqera** [comment]
Looks like this needs a signed commit? I'll have to sort that out somehow.

**Pull request merged**